versioned_docs/version-5/migration.mdx

```diff
@@ -62,6 +62,43 @@ const machine = Machine({
 </TabItem>
 </Tabs>
 
+### Use `createActor()`, not `createActor()`
```

Comment:
I think the global find and replace replaced `interpret()` in the migration instructions!